src/app/wallet/send/send.component.html

```diff
@@ -77,7 +77,7 @@
       <input class="col-xs-4" type="number"
         placeholder="0.00000000"
         [(ngModel)]="send.amount" (input)="verifyAmount()"
-        [ngClass]="{'verify-sucess': checkAmount() === true, 'verify-error': checkAmount() === false }">
+        [ngClass]="{'verify-sucess': checkAmount() === true, 'verify-error': checkAmount() === false }" oninput="validity.valid||(value='');" step="any">
```

Comment:
oninput="validity.valid||(value='');" step="any"

I think this line was copy pasted and they forgot to remove this?